planutils/packages/lama-first/run

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-downward --alias lama-first $@
+planutils run downward -- --alias lama-first $@
```

Comment:
What's the impact on those packages that don't redirect? Works as normal, ya? Should we fix any other alias' that might be affected similarly?

Reply:
Good point. I'll fix the remaining packages that are affected.